ocs_ci/ocs/cluster_load.py

```diff
@@ -0,0 +1,278 @@
+"""
+A module for cluster load related functionalities
+
+"""
+import logging
+import time
+
+import ocs_ci.ocs.constants as constant
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import (
+    CephCluster, get_osd_pods_memory_sum, get_percent_used_capacity
+)
+
+
+class ClusterLoad:
```

Comment:
```suggestion
class ClusterLoad(object):
```

Reply:
I don't think it is necessary. See https://stackoverflow.com/questions/4015417/python-class-inherits-object/9448136